### PR TITLE
Fix minor typo (servers -> serves)

### PR DESCRIPTION
**Pull request by vikassrivastava18**
Please fix this typo.

Thanks for this great tutorial!

### CODE REVIEW

**Review by dipikabh — APPROVED** [review]
Thank you!

**Comment by dipikabh** [comment]
Welcome to the MDN Web Docs project! 🎉
If you're looking for other ways to contribute, we have a [contributor task board](https://github.com/orgs/mdn/projects/25). Issues labeled with `good first issue` are a great place to start.

File: files/en-us/learn/server-side/django/skeleton_website/index.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ The _locallibrary_ project sub-folder is the entry point for the website:
...
 - **urls.py** defines the site URL-to-view mappings. While this could contain _all_ the URL mapping code, it is more common to delegate some of the mappings to particular applications, as you'll see later.
 - **wsgi.py** is used to help your Django application communicate with the web server. You can treat this as boilerplate.
-- **asgi.py** is a standard for Python asynchronous web apps and servers to communicate with each other. ASGI is the asynchronous successor to WSGI and provides a standard for both asynchronous and synchronous Python apps (whereas WSGI provided a standard for synchronous apps only). It is backward-compatible with WSGI and supports multiple servers and application frameworks.
+- **asgi.py** is a standard for Python asynchronous web apps and serves to communicate with each other. ASGI is the asynchronous successor to WSGI and provides a standard for both asynchronous and synchronous Python apps (whereas WSGI provided a standard for synchronous apps only). It is backward-compatible with WSGI and supports multiple servers and application frameworks.
```
Comment (Josh-Cena) [inline files/en-us/learn/server-side/django/skeleton_website/index.md:104]: @dipikabh Could you revert this? This sentence was clearly saying that there are an app (i.e. client) and a server which communicate with each other.

Comment (dipikabh) [reply files/en-us/learn/server-side/django/skeleton_website/index.md:104]: Oops. Yes, let me revert it. Thanks for catching.

Comment (dipikabh) [reply files/en-us/learn/server-side/django/skeleton_website/index.md:104]: Opened https://github.com/mdn/content/pull/25182

**Pull request merged**